driverVersions.json

```diff
@@ -133,6 +133,10 @@
         "66": {
             "driverNPMPackageVersion": "1.16.2",
             "driverVersion": "0.24.0"
+        },
+        "67": {
+            "driverNPMPackageVersion": "1.16.2",
+            "driverVersion": "0.24.0"
```

Comment:
there does not seem to be a newer release of geckodriver for Firefox 67 yet, so using the latest ones.